### PR TITLE
Update FAQs

### PR DESCRIPTION
**Pull request by ShobhitAd**
Add additional FAQs to the existing FAQs page

### CODE REVIEW

File: docs/FAQ/index.md [hunk 1]
```diff
@@ -6,13 +6,34 @@ Here are a few of the most common questions new developers have around the SDL C
...
-```bash
-sudo apt-get update
-sudo apt-get install git cmake build-essential sqlite3 libsqlite3-dev libssl1.0-dev libssl1.0.0 libusb-1.0-0-dev libudev-dev libgtest-dev libbluetooth3 libbluetooth-dev bluez-tools libpulse-dev python3-pip python3-setuptools
+## Can I use SDL on Android OS?
+
+There is no official port at the moment, so individual investigation will need to be done. Even though SDL is designed to work on most Linux systems, modifications might need to be made to the project to get it to work with your setup.
+
+## Why are my RPC requests being DISALLOWED by SDL?
+
+The `DISALLOWED` [result code](https://smartdevicelink.com/en/guides/sdl-overview-guides/rpc-spec/#result) is related to the RPC not being authorized in core's local policy table. Policy permissions for an app are added either in the [preloaded policy table](https://github.com/smartdevicelink/sdl_core/blob/master/src/appMain/sdl_preloaded_pt.json) or through a [policy table update](https://smartdevicelink.com/en/guides/sdl-overview-guides/policies/overview/#policy-table-updates).
```
Comment (JackLivio) [inline docs/FAQ/index.md]:
```suggestion
The `DISALLOWED` [result code](https://smartdevicelink.com/en/guides/sdl-overview-guides/rpc-spec/#result) is related to the RPC not being authorized in SDL Core's local policy table. Policy permissions for an app are added either in the [preloaded policy table](https://github.com/smartdevicelink/sdl_core/blob/master/src/appMain/sdl_preloaded_pt.json) or through a [policy table update](https://smartdevicelink.com/en/guides/sdl-overview-guides/policies/overview/#policy-table-updates).
```

File: docs/FAQ/index.md [hunk 2]
```diff
@@ -6,13 +6,34 @@ Here are a few of the most common questions new developers have around the SDL C
...
+
+The [Policies Overview](https://smartdevicelink.com/en/guides/sdl-overview-guides/policies/overview/) page provides general information about policies - explaining what they are used for, how the policy table gets updated and how these updates are triggered.[Policy Table Fields](https://smartdevicelink.com/en/guides/sdl-overview-guides/policies/policy-fields/) and [App Policies](https://smartdevicelink.com/en/guides/sdl-overview-guides/policies/app-policies/) go into more detail about the policy table structure and how to correctly add policy permissions for an application.
```
Comment (JackLivio) [inline docs/FAQ/index.md]:
```suggestion
The [Policies Overview](https://smartdevicelink.com/en/guides/sdl-overview-guides/policies/overview/) page provides general information about policies - explaining what they are used for, how the policy table gets updated, and how these updates are triggered.[Policy Table Fields](https://smartdevicelink.com/en/guides/sdl-overview-guides/policies/policy-fields/) and [App Policies](https://smartdevicelink.com/en/guides/sdl-overview-guides/policies/app-policies/) go into more detail about the policy table structure and how to correctly add policy permissions for an application.
```

File: docs/FAQ/index.md [hunk 3]
```diff
@@ -6,13 +6,34 @@ Here are a few of the most common questions new developers have around the SDL C
...
+
+## Changes I made to my preloaded policy table aren't reflected in core, what should I do?
+
+If you are not running core for the first time, core will use the existing policy table database(`policy.sqlite`) which is stored in the build folder under `bin/storage/`. To make core use the preloaded policy table again you have to delete the existing policy table database. In the bin folder run:
```
Comment (JackLivio) [inline docs/FAQ/index.md]:
```suggestion
If you are not running SDL Core for the first time, SDL Core will use the existing policy table database(`policy.sqlite`) which is stored in the build folder under `bin/storage/`. To make SDL Core parse the preloaded policy table again you have to delete the existing policy table database. In the bin folder run:
```

File: docs/FAQ/index.md [hunk 4]
```diff
@@ -6,13 +6,34 @@ Here are a few of the most common questions new developers have around the SDL C
...
+
+```shell
+    rm storage/policy.sqlite
 ```
 
+## Can I build SDL with/without certain features(such as logging or build tests)?
+
+You can enable/disable certain features by modifying the [CMakeLists.txt](https://github.com/smartdevicelink/sdl_core/blob/master/CMakeLists.txt) file. The [Cmake Build Configuration section](../getting-started/install-and-run/#cmake-build-configuration) section contains a list of features which can be included/excluded for a build.
```
Comment (JackLivio) [inline docs/FAQ/index.md]:
```suggestion
You can enable/disable certain features by modifying the [CMakeLists.txt](https://github.com/smartdevicelink/sdl_core/blob/master/CMakeLists.txt) file. The [CMake Build Configuration section](../getting-started/install-and-run/#cmake-build-configuration) section contains a list of features which can be included/excluded for a build.
```

File: docs/FAQ/index.md [hunk 5]
```diff
@@ -6,13 +6,34 @@ Here are a few of the most common questions new developers have around the SDL C
 Currently the SDL Core repo is built for Ubuntu 18.04 as our default environment.
 
 ## I'm getting a lot of compilation errors, how do I get past them?
-The most common errors come from dependancies issues. Ensure your system has all the required packages to compile the project. Try running the following commands:
+The most common errors come from dependencies issues. Ensure your system has all the required packages to compile the project. Try running the commands in the [dependencies section](../getting-started/install-and-run/#dependencies) of the getting started guide.
```
Comment (theresalech) [inline docs/FAQ/index.md]:
```suggestion
The most common errors come from dependencies issues. Ensure your system has all the required packages to compile the project. Try running the commands in the [dependencies section](../getting-started/install-and-run/#dependencies) of the Getting Started guide.
```

File: docs/FAQ/index.md (hunk 2 above)
Comment (theresalech) [reply docs/FAQ/index.md]: I updated Jack's suggestion to add a space before "[Policy Table Fields]".  This seemed to break showing the actual suggestions, so I've summarized below:
- Add space after period and before "[Policy Table Fields]"
- Add comma after "...how the policy table gets updated"

File: docs/FAQ/index.md [hunk 6]
```diff
@@ -6,13 +6,34 @@ Here are a few of the most common questions new developers have around the SDL C
...
+
+The `DISALLOWED` [result code](https://smartdevicelink.com/en/guides/sdl-overview-guides/rpc-spec/#result) is related to the RPC not being authorized in core's local policy table. Policy permissions for an app are added either in the [preloaded policy table](https://github.com/smartdevicelink/sdl_core/blob/master/src/appMain/sdl_preloaded_pt.json) or through a [policy table update](https://smartdevicelink.com/en/guides/sdl-overview-guides/policies/overview/#policy-table-updates).
+
+The [Policies Overview](https://smartdevicelink.com/en/guides/sdl-overview-guides/policies/overview/) page provides general information about policies - explaining what they are used for, how the policy table gets updated and how these updates are triggered.[Policy Table Fields](https://smartdevicelink.com/en/guides/sdl-overview-guides/policies/policy-fields/) and [App Policies](https://smartdevicelink.com/en/guides/sdl-overview-guides/policies/app-policies/) go into more detail about the policy table structure and how to correctly add policy permissions for an application.
+
+## Changes I made to my preloaded policy table aren't reflected in core, what should I do?
```
Comment (theresalech) [inline docs/FAQ/index.md]:
```suggestion
## Changes I made to my preloaded policy table aren't reflected in SDL Core; what should I do?
```

File: docs/FAQ/index.md [hunk 7]
```diff
@@ -6,13 +6,34 @@ Here are a few of the most common questions new developers have around the SDL C
...
+
+If you are not running core for the first time, core will use the existing policy table database(`policy.sqlite`) which is stored in the build folder under `bin/storage/`. To make core use the preloaded policy table again you have to delete the existing policy table database. In the bin folder run:
+
+```shell
+    rm storage/policy.sqlite
 ```
 
+## Can I build SDL with/without certain features(such as logging or build tests)?
```
Comment (theresalech) [inline docs/FAQ/index.md]:
```suggestion
## Can I build SDL with/without certain features (such as logging or build tests)?
```

File: docs/FAQ/index.md [hunk 8]
```diff
@@ -6,13 +6,34 @@ Here are a few of the most common questions new developers have around the SDL C
...
+
+You can enable/disable certain features by modifying the [CMakeLists.txt](https://github.com/smartdevicelink/sdl_core/blob/master/CMakeLists.txt) file. The [Cmake Build Configuration section](../getting-started/install-and-run/#cmake-build-configuration) section contains a list of features which can be included/excluded for a build.
+
+## What options can I modify in SDL without having to rebuild?
+
+The [SmartDeviceLink.ini](https://github.com/smartdevicelink/sdl_core/blob/master/src/appMain/smartDeviceLink.ini) file located in your `build/src/appMain` directory is where runtime options can be configured for your instance of SDL Core. The [INI Configuration](../getting-started/ini-configuration) page has more information about individual runtime options.
+
 ## I'm experiencing choppy audio through bluetooth, what should I do?
```
Comment (theresalech) [inline docs/FAQ/index.md]:
```suggestion
## I'm experiencing choppy audio through Bluetooth; what should I do?
```

File: docs/FAQ/index.md (hunk 2 above)
Comment (ShobhitAd) [reply docs/FAQ/index.md]: Fixed in https://github.com/smartdevicelink/sdl_core_guides/pull/70/commits/15752fbe4ed9a55cac24c2459a9bb1b85806114f

**Comment by ShobhitAd** [comment]
https://github.com/smartdevicelink/sdl_core_guides/pull/70#pullrequestreview-368982334 Some of the links are currently broken but, as you noted, they should be fixed those pages in the updated guides are merged into master.

**Pull request merged**